react/tic-tac-toe/src/app/screens/Login/index.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import LoginForm from '../Login/components/LoginForm';
+
+class Login extends Component {
+  submit = (values) => {
+    this.props.checkCredentials(values);
+  };
+
+  render() {
+    if (this.props.userAuthenticated && !this.props.loading) {
+      this.props.history.push("/game");
```

Comment:
This logic shouldn't be here, may be this can be done inside the `PublicRoute` component (similar to `PrivateRoute` behavior)